src/function/table/read_csv.cpp

```diff
@@ -53,6 +53,25 @@ void ReadCSVData::FinalizeRead(ClientContext &context) {
 	}
 }
 
+uint8_t GetCandidateSpecificity(const LogicalType &candidate_type) {
+	//! Const ht with accepted auto_types and their weights in specificity
+	const duckdb::unordered_map<uint8_t, uint8_t> auto_type_candidates_specificity {
```

Comment:
Do we need to constrain this or re-order this for the user? I would say this is relatively advanced usage already so if the user puts a `TINYINT` after an `INTEGER` then I would say that's user error.

Reply:
Hm, I think that we would either do nothing, then the user might get an unexpected result. We could throw an error, but at that point, we are already doing the checking, so it seems that reordering is just convenient, no?